apps/webapp/webpack.config.js

```diff
@@ -70,6 +70,9 @@ module.exports = {
         'style-loader?sourceMap',
         'css?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]'
       )
+    },{
+      test: /\.json$/,
```

Comment:
This loader only works if you have files you need to load during assets compile time. That said, you need/are _fetching_ the json dynamically using **fetch**